Documentation/contributing/testing/e2e.rst

```diff
@@ -287,6 +287,30 @@ needing to run the full test, you can build the test directory:
 
     make -C test/ build
 
+Updating Cilium images for Kubernetes tests
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes when running the CI suite for a feature under development, it's common
+to re-run the CI suite on the CI VMs running on a local development machine after
+applying some changes to Cilium. For this the new Cilium images have to be
+built, and then used by the CI suite. To do so, one can run the following
+commands on the ``k8s1`` VM:
+
+.. code-block:: shell-session
+
+   cd go/src/github.com/cilium/cilium
+
+   make LOCKDEBUG=1 docker-cilium-image
+   docker tag quay.io/cilium/cilium:latest k8s1:5000/cilium/cilium-dev
+   docker push k8s1:5000/cilium/cilium-dev
```

Comment:
Should this image be tagged as `latest` too?

```suggestion
   docker tag quay.io/cilium/cilium:latest k8s1:5000/cilium/cilium-dev:latest
   docker push k8s1:5000/cilium/cilium-dev:latest
```

Reply:
Yep (although not specifying it defaults to `latest`).